job-service-db/src/main/resources/procedures/getJobs.sql

```diff
@@ -145,8 +145,7 @@ BEGIN
         sql := sql || whereOrAnd || in_filter;
     END IF;
 
-    sql := sql || ' ORDER BY ' || quote_ident(in_sort_field) ||
-        ' ' || CASE WHEN in_sort_ascending THEN 'ASC' ELSE 'DESC' END;
+    sql := sql || ' ORDER BY create_date ' || CASE WHEN in_sort_ascending THEN 'ASC' ELSE 'DESC' END;
```

Comment:
This change isn't correct because when you combine it with the limit it will eliminate rows that shouldn't be eliminated.